userspace/libsinsp/test/dns_manager.ut.cpp

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2023 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include <gtest/gtest.h>
+#include "sinsp_with_test_input.h"
+#include <dns_manager.h>
```

Comment:
The rules for ordering the header files vary a lot from organization to organization. However, there is some consensus in the industry that the headers "closest" to the current compilation unit should go first, then more distant ones such as 3rd party libs and, finally, standard header files. The main reason is that including a system/library header file _before_ the local ones might hide the fact that they miss a critical header for their internal definition, only to be sadly discovered by someone, at some later point in time, who uses these headers in a different context.
According to this idea GTest header should go last in this case.
It's your call whether you feel to embrace this.
As an aside, I think the ordering of headers should be put in the project's coding guidelines.